apis/theme/src/style-resolver.js

```diff
@@ -94,18 +106,22 @@ export default function styleResolver(basePath, themeJSON) {
      * object.barChart - fontSize
      * object - fontSize
      * fontSize
+     * When defaultValue is provided,
+     * it get the value of a style attribute with the given base path + path
+     * Ex: object.barChart - legend.title - fontSize
```

Comment:
This doesn't really make sense, something like this instead:
```suggestion
     * it will fallback to defaultValue instead of searching through the tree
```

Reply:
it gets the value of a style attribute with the given base path + pat, if not found, it will fallback to defaultValue

Reply:
yupp that is better